build.sh

```diff
@@ -113,3 +124,27 @@ else
     fi
 fi
 popd > /dev/null # ${FRAMEWORK_DIR}
+
+
+CMAKE_BIN=${UT_CONTROL_LIB_DIR}/host-tools/CMake-3.30.0/build/bin/cmake
+if [ ! -f "$CMAKE_BIN" ]; then
+    CMAKE_BIN=$CMAKE_VAR
+fi
+
+pushd ${MY_DIR} > /dev/null
+#Clone GTEST
+if [[ ! -d "${GTEST_DIR}/googletest-1.15.2" && "${VARIANT}" == "CPP" ]]; then
+    wget https://github.com/google/googletest/archive/refs/tags/v1.15.2.zip --no-check-certificate -P ${GTEST_DIR}
```

Comment:
need to set the version externally do this so we can control all the version together.